test/index.test.js

```diff
@@ -0,0 +1,616 @@
+        const c = 2;
+        const expected = 14;
+
+        const actual = getMax(a, b, c);
+
+        assert.deepEqual(actual, expected);
+    });
+
+    it('should be return 0 (undefined, undefined, undefined)', () => {
+        const a = undefined;
+        const b = undefined;
+        const c = undefined;
+        const expected = 0;
+
+        const actual = getMax(a, b, c);
+
+        assert.deepEqual(actual, expected);
+    });
+});
+
+describe('getMark', () => {
+    it('should be return A (91)', () => {
+        const rate = 91;
+        const expected = 'A';
+
+        const actual = getMark(rate);
+
+        assert.deepEqual(actual, expected);
+    });
+
+    it('should be return F (1)', () => {
+        const rate = 1;
+        const expected = 'F';
+
+        const actual = getMark(rate);
+
+        assert.deepEqual(actual, expected);
+    });
+
+    it('should be return C (74)', () => {
+        const rate = 74;
+        const expected = 'C';
+
+        const actual = getMark(rate);
+
+        assert.deepEqual(actual, expected);
+    });
+
+    it('should be return empty line (-1)', () => {
+        const rate = -1;
+        const expected = '';
+
+        const actual = getMark(rate);
+
+        assert.deepEqual(actual, expected);
+    });
+
+    it('should be return empty line (101)', () => {
+        const rate = 101;
+        const expected = '';
+
+        const actual = getMark(rate);
+
+        assert.deepEqual(actual, expected);
+    });
+
+
+    it('should be return empty line (undefined)', () => {
+        const rate = undefined;
+        const expected = '';
+
+        const actual = getMark(rate);
+
+        assert.deepEqual(actual, expected);
+    });
+});
+
+describe('getEvenSum', () => {
+    it('should be return 2450', () => {
+        const expected = 2450;
+        const actual = getEvenSum();
+
+        assert.deepEqual(actual, expected);
+    });
+});
+
+describe('getIsPrime', () => {
+    it('should be return true (2)', () => {
+        const num = 2;
+        const expected = true;
+        const actual = getIsPrime(num);
+
+        assert.deepEqual(actual, expected);
+    });
+
+    it('should be return true (3)', () => {
+        const num = 3;
+        const expected = true;
+        const actual = getIsPrime(num);
+
+        assert.deepEqual(actual, expected);
+    });
+
+    it('should be return false (8)', () => {
+        const num = 8;
+        const expected = false;
+        const actual = getIsPrime(num);
+
+        assert.deepEqual(actual, expected);
+    });
+
+    it('should be return false (-1)', () => {
+        const num = -1;
+        const expected = false;
+        const actual = getIsPrime(num);
+
+        assert.deepEqual(actual, expected);
+    });
+
+    it('should be return undefined (undefined)', () => {
+        const num = undefined;
+        const expected = undefined;
+        const actual = getIsPrime(num);
+
+        assert.deepEqual(actual, expected);
+    });
```

Comment:
what it returns for 9?
what it returns for 25?
what it returns for 26?
Please, read carefully what the prime number is